### PR TITLE
Testing with flutter_lints DartPad

### PR DESCRIPTION
**Pull request by sfshaza2**
Staged: https://sz-flutter.firebaseapp.com/

DO NOT MERGE. This is a test of the null safe DartPad.

### CODE REVIEW

File: src/docs/codelabs/implicit-animations.md [hunk 1]
```diff
@@ -62,10 +62,10 @@ Click the **Run** button to run the example:
 {% include implicit-animations/fade-in-starter-code.md %}
 
 {{site.alert.important}}
-  This page uses an embedded version of [DartPad] to display
+  This page uses an embedded version of [DartPad][] to display
```
Comment (domesticmouse) [inline src/docs/codelabs/implicit-animations.md:65]: These fixes should probably be pulled out into a separate PR and actually landed

Comment (sfshaza2) [reply src/docs/codelabs/implicit-animations.md:65]: That's what Andrew was doing. He just asked Kathy and I to test it. Closing this PR.
